resources/js/Pages/Maintenance.vue

```diff
@@ -1,189 +1,65 @@
-                        href="https://www.facebook.com/NIAEFEUP"
+                <div class="mx-auto md:ml-36 flex-grow md:flex-grow-0">
+                    <h1
+                        class="w-3/12 text-justify font-space-grotesk text-5xl font-bold tracking-tight text-white md:text-7xl"
                     >
-                        <OhVueIcon
-                            name="io-logo-facebook"
-                            fill="white"
-                        ></OhVueIcon>
-                    </a>
-                    <a
-                        class="flex w-fit rounded-full bg-2023-red-dark p-2"
-                        target="_blank"
-                        href="https://www.linkedin.com/company/nifeup/mycompany/"
-                    >
-                        <OhVueIcon
-                            name="io-logo-linkedin"
-                            fill="white"
-                        ></OhVueIcon>
-                    </a>
+                        <p class="whitespace-nowrap lowercase">Encontro</p>
+                        <p class="whitespace-nowrap lowercase">Nacional de</p>
+                        <p class="whitespace-nowrap lowercase">Estudantes de</p>
+                        <p class="whitespace-nowrap lowercase">Informática</p>
+                    </h1>
                     <p
-                        class="text-lg font-bold text-2023-teal max-[900px]:text-base"
+                        class="font-space-grotesk text-2xl text-white md:text-4xl uppercase"
                     >
-                        externas@sinf.pt
+                        Porto 2025
                     </p>
-                    <a
-                        class="flex w-fit rounded-full bg-2023-teal p-2"
-                        target="_blank"
-                        href="https://www.instagram.com/niaefeup/"
-                    >
-                        <OhVueIcon
-                            name="io-logo-instagram"
-                            fill="white"
-                        ></OhVueIcon>
-                    </a>
-                    <a
-                        class="flex w-fit rounded-full bg-2023-orange p-2"
-                        target="_blank"
-                        href="https://twitter.com/niaefeup"
-                    >
-                        <OhVueIcon
-                            name="io-logo-twitter"
-                            fill="white"
-                        ></OhVueIcon>
-                    </a>
                 </div>
-                <div
-                    id="contacts-content-mobile"
-                    class="hidden flex-col items-center gap-2.5 max-ml:flex"
-                >
+                <div class="mx-auto">
                     <p
-                        class="text-lg font-bold text-2023-teal max-ml:text-base"
+                        class="mr-2 w-48 bg-2023-bg p-2.5 px-8 text-center font-space-grotesk text-lg text-lg font-bold font-bold text-enei-blue uppercase"
```

Comment:
What's `bg-2023-bg`?

Reply:
In the `tailwind.config.js` file there is a color called `2023-bg` which is the sinf beige from the sinf website. 

Because the enei and sinf beige look extremely similar that was a detail we overlooked. It should be `bg-enei-beige`